app-backend/api/src/main/scala/Router.scala

```diff
@@ -62,6 +63,7 @@ trait Router extends HealthCheckRoutes
     } ~
     pathPrefix("api") {
       pathPrefix("projects") { projectRoutes } ~
+      pathPrefix("aoi") { aoiRoutes } ~
```

Comment:
Ah - I think this endpoint should be `areas-of-interest` instead of `aoi` so that it's plural and more user-friendly.